jobs/keepalived/templates/bin/post-start.erb

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+	
+#set -e # exit immediately if a simple command exits with a non-zero status
```

Comment:
why this is disabled ?

Reply:
assumed that if any resource is not present, it will throw an error and exit. This shouldn't be the case, though, if any resource is not present, we should create it.

Reply:
bt I checked.. setting it as -e doesn't quit the process, if the resource is not present. So will add it back.